src/go/rpk/pkg/cli/transform/buildpack/buildpack.go

```diff
@@ -95,13 +99,15 @@ func (bp *Buildpack) PlatformSha() (sha string, err error) {
 
 // URL is the fully resolved URL to download the gzipped tarball for the current platform.
 func (bp *Buildpack) URL() string {
-	return fmt.Sprintf(
-		"%s/%s-%s-%s.tar.gz",
-		bp.baseURL,
-		bp.Name,
-		runtime.GOOS,
-		runtime.GOARCH,
-	)
+	t := template.Must(template.New("URL").Parse(bp.template))
+	b := bytes.NewBuffer(nil)
+	template.Must(t, t.Execute(b, map[string]string{
+		"Name":    bp.Name,
+		"BaseURL": bp.baseURL,
```

Comment:
q: the "Name" field is quietly ignored if it doesn't appear in the base template then?

Reply:
yes exactly